### PR TITLE
Ca doc

### PR DESCRIPTION
**Pull request by cauvigne**
- Edit the different README and CONTRIBUTING with installation observation
- Add paypal link taken from soaringmeteo.org **Warning: the button links to a 2018 donation**
- Add issue templates
-  Remove deployment to soaringmeteo server since it should be reserved to soaringmeteo.org...???

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: BUG
+labels: bug
+---
+
+**Commit version**
+
+The commit version goes here.
```
Comment (julienrf) [inline .github/ISSUE_TEMPLATE/bug-report.md:10]: I am not sure the users can easily tell which commit version they use, should we remove this part?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,21 +1,129 @@
-# Contributing
+# How to contribute
 
-## Evolving the Format of the Forecast Data
+Soaringmeteo team is glad you are reading this, because we always need volunteers
+to keep this awesome service up to date.
```
Comment (julienrf) [inline CONTRIBUTING.md:4]:
```suggestion
to keep this service up to date.
```

I think we should adopt a more neutral tone.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,21 +1,129 @@
-# Contributing
+# How to contribute
 
-## Evolving the Format of the Forecast Data
```
Comment (julienrf) [inline CONTRIBUTING.md:3]: I would like to keep this section and the content that was in it. Could you please add it back somewhere?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,21 +1,129 @@
...
+Soaringmeteo team is glad you are reading this, because we always need volunteers
+to keep this awesome service up to date.
 
-The frontend consumes the forecast data produced by the backend, so both parties need to agree on their format.
-Furthermore, the frontend also reads several forecast data in the past (users can show the data from the latest
-forecast run, or from older runs as well). This has consequences on how the format of the forecast data produced
-by the backend can evolve:
+* Find a bug? Do not hesitate to open an issue using the bug template. **TODO: LINK HERE**
+* Have an awesome improvement idea? Do not hesitate to open an issue using the feature template **TODO: LINK HERE**
```
Comment (julienrf) [inline CONTRIBUTING.md:7]: I think the link should be: https://github.com/soaringmeteo/soaringmeteo/issues/new/choose

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,21 +1,129 @@
...
+gfs/Universal/packageZipTarball
+tar -xzf soaringmeteo-gfs.tgz
+```
+
+4. Build the `wrf` program:
+
+```
+wrf/Universal/packageZipTarball
+tar -xzf soaringmeteo-wrf.tgz
+```
```
Comment (julienrf) [inline CONTRIBUTING.md:76]: Actually, this step is not necessary, the tasks `makeGfsAssets` and `makeWrfAssets` automatically compile the program before running it.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,21 +1,129 @@
...
+	makeGfsAssets
+	makeWrfAssets
+	```
+
+The different backend options are documented [here](backend/README.md)
+
+# Run the frontend
+
+```
+yarn install --immutable
```
Comment (julienrf) [inline CONTRIBUTING.md:94]: Maybe add that the user should change the working directory to `frontend`?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,21 +1,129 @@
...
+```
+
+Open a browser to the address suggested in your console (by default `127.0.0.1:3000`).
+Congratulations! Your development environment is ready.
+
+# Test the code
+
+## Backend
+
+Open a console in the `backend folder` and run the `sbt` tool to get a `sbt` console.
```
Comment (julienrf) [inline CONTRIBUTING.md:106]:
```suggestion
Open a console in the `backend` folder and run the `sbt` tool to get a `sbt` console.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,21 +1,129 @@
...
+The following commands run all the available tests:
```
Comment (julienrf) [inline CONTRIBUTING.md:107]:
```suggestion
The following command runs all the available tests:
```

File: README.md [hunk 9]
```diff
@@ -23,21 +23,40 @@ This repository contains two sub-projects:
 - [backend](backend/), which produces the forecast data,
 - [frontend](frontend/), which displays the forecast data.
 
-Please refer to each sub-project for more details.
 
-## Contributing
+The backend part is made of three distinct programs:
+
+- `gfs`, the GFS pipeline, which process data from the GFS model
```
Comment (julienrf) [inline README.md:29]:
```suggestion
- `gfs`, the GFS pipeline, which processes data from the GFS model
```

File: README.md [hunk 10]
```diff
@@ -23,21 +23,40 @@ This repository contains two sub-projects:
...
+- `wrf`, the WRF pipeline, which process data from the WRF model
```
Comment (julienrf) [inline README.md:30]:
```suggestion
- `wrf`, the WRF pipeline, which processes data from the WRF model
```

File: frontend/README.md [hunk 11]
```diff
@@ -1,39 +1,14 @@
...
+
+Frontend that consumes the JSON files produced by the backend.
+
+# Setup
+
+See the [CONTRIBUTING.md](../CONTRIBUTING.md) file.
+
+# Code conventions
+
+# Test the code
```
Comment (julienrf) [inline frontend/README.md:11]: Should we remove those empty sections?

**Review by julienrf — CHANGES_REQUESTED** [review]
Thank you for improving the instructions @cauvigne! Here is my review.

Also, should we add another issue template for feature requests?

File: CONTRIBUTING.md (hunk 3 above)
Comment (cauvigne) [reply CONTRIBUTING.md:3]: Surething, done in CONTRIBUTING.md

File: CONTRIBUTING.md (hunk 2 above)
Comment (cauvigne) [reply CONTRIBUTING.md:4]: done

File: CONTRIBUTING.md (hunk 6 above)
Comment (cauvigne) [reply CONTRIBUTING.md:104]: done

**Pull request merged**